content/cloud/deployment-concepts.md

```diff
@@ -8,38 +8,54 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/deployment-c
 ---
 
 - [Deployments in Fermyon Cloud](#deployments-in-fermyon-cloud)
-- [OCI - the Open Container Initiative](#oci---the-open-container-initiative)
+- [Bindle - An Aggregate Object Storage System](#bindle---an-aggregate-object-storage-system)
```

Comment:
Looks like this this document deployment-concepts.md is being reverted. Can we undo the changes here?

Reply:
Kate fixed it!